saleor/product/models/base.py

```diff
@@ -370,3 +370,13 @@ class AttributeChoiceValue(models.Model):
 
     def __str__(self):
         return self.display
+
+
+@python_2_unicode_compatible
+class Collection(models.Model):
+    name = models.CharField(
+        pgettext_lazy('Collection field', 'name'), max_length=128, unique=True)
```

Comment:
I think `Product collection field` would make it clearer. It's supposed to provide enough context for translators.